ios/RNOneginiSdk.swift

```diff
@@ -176,7 +176,7 @@ class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
     }
 
     @objc
-    func authenticateUser(_ profileId: (NSString),
+    func authenticateUser(_ profileId: (NSString), authenticatorId authenticator: (NSString),
```

Comment:
Should be rather `NSString *`

Reply:
Instead I replaced all NSString references with swift String.